src/main/java/com/redhat/pantheon/data/ModuleDataRetriever.java

```diff
@@ -26,6 +26,19 @@ public ModuleDataRetriever(ResourceResolver resolver) {
         this.resolver = resolver;
     }
 
+	public List<Map<String, Object>> getModulesSort(String searchTerm, String key, String direction) {
+		if (searchTerm == null || searchTerm.isEmpty()) {
+			searchTerm.equals("*");
+		}
+		if (key == null || (!key.equals("jcr:title") && !key.equals("jcr:description"))) {
```

Comment:
Looks good! In the future, don't forget you can avoid the null check via `!"jcr:title".equals(key)` instead. :)

Reply:
thanks for pointing that out I'll do that next time.